### ISSUE TITLE
Getting RateLimited

### ISSUE THREAD

**Issue by tinaenriquez** [CircleCI-Public/CircleCI-Env-Inspector#5]
Getting this error after inputs

```
undefined:1
Rate limited.
^

SyntaxError: Unexpected token 'R', "Rate limited." is not valid JSON
    at JSON.parse (<anonymous>)
    at Response.json (file:///project/node_modules/node-fetch/src/body.js:149:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///project/index.js:139:21
    at async Promise.all (index 252)
    at async file:///project/index.js:136:18

Node.js v19.3.0
```

**Comment by shubhamshukla89** [comment]
Getting the same issue, did this but didnt resolve. @EricRibeiro

**Comment by jtheuer** [comment]
Same issue here. Can you provide a fix @EricRibeiro ?

**Comment by aaroncarson-monzo** [comment]
Me too

**Comment by searnshaw-gresham** [comment]
+1, seeing the same behaviour after adding the suggested fix.

**Comment by tdeekens** [comment]
I also tried by now with a 10s delay in-between any request. Same issue of being rate limited.

**Comment by KyleTryon** [comment]
Hey folks, just coming in for the day. Myself and @EricRibeiro will get this fixed asap

**Comment by afilbert** [comment]
I'm still seeing the same as well.

**Comment by EricRibeiro** [comment]
This should be fixed with the merge of #14. Please try again and let us know.

**Comment by afilbert** [comment]
> This should be fixed with the merge of #14. Please try again and let us know.

Pulled latest and it's working great for us, thank you!

**Comment by KyleTryon** [comment]
Thanks for confirming @afilbert. Going to close for now but if anyone is still experiencing this issue let us know and we'll re-open.
